enterprise/admin/__init__.py

```diff
@@ -1332,8 +1358,7 @@ class DefaultEnterpriseEnrollmentIntentionAdmin(admin.ModelAdmin):
 
     readonly_fields = (
         'current_course_run_key',
-        'current_course_run_enrollable',
-        'current_course_run_enroll_by_date',
```

Comment:
It was decided for the moment, since the properties on the model are yet to be fleshed out, these fields should be removed.